custom_components/meross_cloud/climate.py

```diff
@@ -3,7 +3,7 @@
 
 from homeassistant.components.climate import (SUPPORT_PRESET_MODE,
                                               SUPPORT_TARGET_TEMPERATURE,
-                                              ClimateDevice)
+                                              ClimateEntity)
 from homeassistant.components.climate.const import (CURRENT_HVAC_HEAT,
```

Comment:
Be Carefull with this, if you just apply this change the component will stop working with previous version of HA.
Change should be somenthing like these (for all import)

```
try:
    from homeassistant.components.climate import ClimateEntity
except ImportError:
    from homeassistant.components.climate import ClimateDevice as ClimateEntity
```